op-e2e/actions/proofs/jovian_activation_test.go

```diff
@@ -0,0 +1,130 @@
+package proofs
+
+import (
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-chain-ops/genesis"
+	actionsHelpers "github.com/ethereum-optimism/optimism/op-e2e/actions/helpers"
+	"github.com/ethereum-optimism/optimism/op-e2e/actions/proofs/helpers"
+	"github.com/ethereum-optimism/optimism/op-program/client/claim"
+	"github.com/ethereum-optimism/optimism/op-service/testlog"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_ProgramAction_JovianActivation(gt *testing.T) {
+
+	runJovianDerivationTest := func(gt *testing.T, testCfg *helpers.TestCfg[any]) {
+		t := actionsHelpers.NewDefaultTesting(gt)
+
+		// Define override to activate Jovian 14 seconds after genesis
+		var setJovianTime = func(dc *genesis.DeployConfig) {
+			// Set all predecessor forks at genesis (0 offset)
+			zero := hexutil.Uint64(0)
+			dc.L2GenesisHoloceneTimeOffset = &zero
+			// Set Isthmus at 10s (required predecessor fork for Jovian)
+			ten := hexutil.Uint64(10)
+			dc.L2GenesisIsthmusTimeOffset = &ten
```

Comment:
Curious why we don't just activate Isthmus at genesis instead?

Reply:
so i tried to start Isthmus at genesis or some time after genesis, and i'm getting this error:

`UpgradeScheduleDeployConfig failed checks: fork isthmus set (to 10), but prior fork holocene missing`